client/src/containers/HomePage/HomePage.jsx

```diff
@@ -172,6 +183,8 @@ class HomePage extends Component {
       || (index == 6 && this.chartsRef.current.state.dialogOpen)
       || (index == 7 && this.chartsRef.current.state.currentDisplay != 5)
       || (index == 8 && this.state.currentDisplay === JOYRIDE_LOCK_STATE)
+      || (index == 9 && this.state.zoom < 6)
+      || (index == 11 && this.state.zoom >= 6)
```

Comment:
new checks to see if the user zoomed out on the zoom steps

Reply:
I think it's step 11 where I click on Hawaii then click on next, and it goes back a step (to the Zoom Out). It works fine if I click on Alaska or Continental

Reply:
Gotcha, I need to add a check to see if it's on the 'step-before' event of the joyride. Hawaii is technically zoomed in enough for counties to render and that one line catches it and sends it back to the step before, changing it now

Reply:
Fixed